bundles/org.openhab.binding.hue/README.md

```diff
@@ -71,9 +86,11 @@ CLIP Sensor set or get by JSON through IP.
 
 Finally, the Hue binding also supports the groups of lights and rooms set up on the Hue Bridge.
 
-## Discovery
+## Bridge Discovery (CLIP v2 and CLIP v1)
```

Comment:
This change could be reverted for simplicity.

Reply:
Ok.